lib/trento/clusters/cluster.ex

```diff
@@ -605,19 +605,20 @@ defmodule Trento.Clusters.Cluster do
 
   defp maybe_emit_cluster_deregistered_event(_, _), do: nil
 
-  defp maybe_add_checks_health(healths, _, []), do: healths
-  defp maybe_add_checks_health(healths, checks_health, _), do: [checks_health | healths]
+  defp maybe_add_checks_health(healths, checks_health) when checks_health != Health.unknown(),
+    do: [checks_health | healths]
+
+  defp maybe_add_checks_health(healths, _), do: healths
```

Comment:
I read this as: take into account `checks_health` when it is anything different from `unknown`, in all other cases (the `_` blackhole in the second match) only consider `healths`

The point is that _all other cases_ in the second match is really _just one case_: when it is `unknown`

What about the following?
```elixir
defp maybe_add_checks_health(healths, Health.unknown()), do: healths
defp maybe_add_checks_health(healths, checks_health), do: [checks_health | healths]
```
or, if we want to have it clearer in the first match what the second argument is
```elixir
defp maybe_add_checks_health(healths, checks_health) when checks_health == Health.unknown(), do: healths
defp maybe_add_checks_health(healths, checks_health), do: [checks_health | healths]
```

Reply:
ah yes, I like this way much more :+1: